plugins/lookup/conjur_variable.py

```diff
@@ -134,24 +137,30 @@ def _load_identity_from_file(identity_path, appliance_url):
 # Merge multiple dictionaries by using dict.update mechanism
 def _merge_dictionaries(*arg):
     ret = {}
-    for a in arg:
-        ret.update(a)
+    for item in arg:
+        ret.update(item)
     return ret
 
 # The `quote` method's default value for `safe` is '/' so it doesn't encode slashes
 # into "%2F" which is what the Conjur server expects. Thus, we need to use this
 # method with no safe characters. We can't use the method `quote_plus` (which encodes
 # slashes correctly) because it encodes spaces into the character '+' instead of "%20"
 # as expected by the Conjur server
-def _encode_str(str):
-    return quote(str, safe='')
+def _encode_str(input_str):
+    return quote(input_str, safe='')
 
 # Use credentials to retrieve temporary authorization token
-def _fetch_conjur_token(conjur_url, account, username, api_key, validate_certs):
+def _fetch_conjur_token(conjur_url, account, username, api_key, validate_certs, cert_file):
```

Comment:
not sure how python works - does this method still work if less than this number of arguments is supplied, or is this a backwards-incompatible change?

Reply:
Technically it's a backwards-incompatible change but this is a private method (`_` prefix) so there should not be anything other than this module using this method.